RockLib.Messaging.SQS/SQSReceiver.cs

```diff
@@ -405,39 +425,43 @@ protected override void Dispose(bool disposing)
         {
             _stopped = true;
             if (_receiveMessagesTask.IsValueCreated)
+            {
                 try { _receiveMessagesTask.Value.Wait(); }
+#pragma warning disable CA1031 // Do not catch general exception types
                 catch { }
+#pragma warning restore CA1031 // Do not catch general exception types
+            }
             SQSClient.Dispose();
             base.Dispose(disposing);
         }
 
         private static string GetAdditionalInformation(AmazonWebServiceResponse response, string receiptHandle)
```

Comment:
Since `receiptHandle` can be null, the parameter type should be `string?`, and then one place where you're calling it, you don't have to do `null!` anymore.